src/models/countries.js

```diff
@@ -0,0 +1,46 @@
+module.exports = (sequelize, DataTypes) => {
+  const Country = sequelize.define("Country", {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      allowNull: false,
+      primaryKey: true,
+    },
+    nameOfCountry: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    gallery: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    capital: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    population: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    officialLanguage: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    region: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    currency: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  });
+
+  Country.associate = (models) => {
+    Country.hasMany(models.Culture, {
```

Comment:
improve your associations and add the `as` and `foreignKey` keyword

Reply:
done

Reply:
improve your hasmany associations to something like this;

for example:
```
Accommodation.hasMany(models.Rooms, { as: 'roomAccommodation', foreignKey: 'accommodationId', onDelete: 'cascade', hooks: true });
```